client/login/wp-login/login-links.jsx

```diff
@@ -350,9 +349,8 @@ export default connect(
 		isLoggedIn: Boolean( getCurrentUserId( state ) ),
 		oauth2Client: getCurrentOAuth2Client( state ),
 		query: getCurrentQueryArguments( state ),
-		isJetpackWooCommerceFlow:
-			'woocommerce-onboarding' === get( getCurrentQueryArguments( state ), 'from' ),
-		wccomFrom: get( getCurrentQueryArguments( state ), 'wccom-from' ),
+		isJetpackWooCommerceFlow: 'woocommerce-onboarding' === getCurrentQueryArguments( state ).from,
```

Comment:
`getCurrentQueryArguments( state )` selector could return `null`, and accessing its `.from` would trigger an error. Should we use optional changing to prevent that?
```suggestion
		isJetpackWooCommerceFlow: 'woocommerce-onboarding' === getCurrentQueryArguments( state )?.from,
```

Reply:
The selector will return either `false` or an object, never `null`. And field access on `false` is possible.

We already use direct `getCurrentQueryArguments( state ).email_address` at several places in Login and it's safe.

I can add `?.from` but we should rather migrate towards state where it's not needed. After all, the app is always on _some_ route, so there's no reason why the current path or query should be `null`.

Reply:
> The selector will return either false or an object, never null. 

If that's the case, should we update the selector? It currently defaults to `null`: 

https://github.com/Automattic/wp-calypso/blob/3eb87c8860b8f24db1d592040af747688dc575e7/client/state/selectors/get-current-query-arguments.js#L11

> We already use direct getCurrentQueryArguments( state ).email_address at several places in Login and it's safe.

That's fair 👍 

> I can add ?.from but we should rather migrate towards state where it's not needed. After all, the app is always on some route, so there's no reason why the current path or query should be null.

Agreed. I'm not sure why the selector defaults to `null` TBH 🤷